### ISSUE TITLE
train and test size of tusimple

### ISSUE THREAD

**Issue by andreysher** [cfzd/Ultra-Fast-Lane-Detection-v2#88]
Hello, and thanks for your job! Can you please explain why train_gt.txt for tusimple dataset have 617 rows, and test.txt have 2782 rows. There is information from official tusimple github: https://github.com/TuSimple/tusimple-benchmark/tree/master/doc/lane_detection about 3626 train annotated images.

**Comment by andreysher** [comment]
I follow the instruction from Readme file, and get 0.95 F1 score instead of 0.96 for resnet18 on tusimple dataset. Can you check the hyper parameters and help me to reproduce your results? Thanks for advice/

**Comment by cfzd** [comment]
@andreysher 
For the first question, I don't know what's the meaning of "rows". The whole dataset of tusimple is like:
```
$TUSIMPLE
|──clips
|──label_data_0313.json
|──label_data_0531.json
|──label_data_0601.json
|──test_tasks_0627.json
|──test_label.json
|──readme.md
```
The files `label_data_0313.json`, `label_data_0531.json`, `label_data_0601.json` are the training set, and `test_tasks_0627.json` is the test set. We use the official setting as tusimple.

For the performance, the shared trained models are trained with the provided corresponding configs exactly. The training is random, you can try more times.
